python_modules/dagster/dagster/_core/definitions/freshness_checks/non_partitioned.py

```diff
@@ -30,6 +30,7 @@ def build_freshness_checks_for_non_partitioned_assets(
     assets: Sequence[Union[CoercibleToAssetKey, AssetsDefinition, SourceAsset]],
     maximum_lag_minutes: int,
     freshness_cron: Optional[str] = None,
+    timezone: Optional[str] = None,
```

Comment:
Maybe I'm anchored on what we did with `FreshnessPolicy`, but I think calling this `cron_timezone` or `freshness_cron_timezone` would make it more clear what this is for.